recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ test:
     - pytest
```

Comment:
1. It is unclear why `git` is a requirement for testing.
2. `pip` should only be used for `pip check` at testing the stage. We have mostly just agreed that we want to "play nicely" with pip so as maintainers of packages, we have found that users call `pip check` to ensure consistency of their environment, and sometimes conda packages need to be re-adjusted.

Reply:
These are also good points. I believe removing git caused an issue, but I can't remember why. I'll try removing it and then add a comment explaining why we need it (if we do).

Yes good point about pip check. I'll try to reintroduce it here after a `pip install thop`, since this is the only python package not available in conda, and the `ultralytics` package includes it in requirements but only uses it if installed.

Reply:
at conda-forge, we don't allow external dependencies from `pip`. adding a `pip install thop` would be against our guidelines.

Your options are:

1. Ignore the `pip check` failure if any
2. Patch out the requirement from upstream. for optional requirements this is typically OK.
3. Add the package to conda-forge.